src/udev/udev-builtin.h

```diff
@@ -37,6 +37,10 @@ typedef struct UdevBuiltin {
         bool run_once;
 } UdevBuiltin;
 
+#define UDEV_NAME_SIZE   512
+#define UDEV_PATH_SIZE  1024
+#define UDEV_LINE_SIZE 16384
```

Comment:
isn't that something to move src/shared/udev-util.h, too? It doesn't really have anything to do with builtins, no?